docs/pages/guides/typescript.mdx

```diff
@@ -0,0 +1,83 @@
+import PageMeta from '_shared/PageMeta';
+
+<PageMeta
+  title="Typescript – @material-ui/pickers"
+  description="A guide of how to use @material-ui/pickers with typescript"
+/>
+
+## Typescript guide
```

Comment:
As far as I know, these capitalizations are not correct: "Typescript", "typescript" and the only valid one is "TypeScript": http://typescriptlang.org/.

Reply:
cc @dtassone I have seen the same in the data grid.